test/unit/serialization_sb2.js

```diff
@@ -15,7 +15,7 @@ test('spec', t => {
 test('default', t => {
     // Get SB2 JSON (string)
     const uri = path.resolve(__dirname, '../fixtures/default.sb2');
-    const file = readAsString(uri);
+    const file = extractProjectJson(uri);
     const json = JSON.parse(file);
```

Comment:
just looking at this diff made me realize that I think every time you use this you call JSON.parse on it, you might as well make that part of the method too?